src/Nelmio/Alice/Fixtures/Builder/Methods/ListName.php

```diff
@@ -25,21 +25,57 @@ class ListName implements MethodInterface
      */
     public function canBuild($name)
     {
-        return 1 === preg_match('#\{([^,]+(\s*,\s*[^,]+)*)\}#', $name, $this->matches);
+        if (1 !== preg_match('/\{(?<content>.*)\}/', $name, $this->matches)) {
```

Comment:
I feel like too much is happening here - what about just breaking this class into two?